packages/medusa/src/services/fulfillment-provider.js

```diff
@@ -0,0 +1,27 @@
+import { MedusaError } from "medusa-core-utils"
+
+/**
+ * Helps retrive fulfillment providers
+ */
+class FulfillmentProviderService {
+  constructor(container) {
+    /** @private {logger} */
```

Comment:
container_

Reply:
I do use `this.container_` which is the standard for private instance variables. The constructor parameters don't need the underscore